### PR TITLE
About page(css edited)

### PR DESCRIPTION
**Pull request by ghost**
Fixed some format of AboutPage, renamed style className in AboutPage.css.

Merge this asap so other people will not get affected by my old aboutpage styles.

### CODE REVIEW

File: frontend/src/Components/AboutPage.js [hunk 1]
```diff
@@ -9,65 +9,71 @@ class AboutPage extends React.Component {
...
-        <h1>About Us</h1>
+        <h1 className="h1-about">About Us</h1>
 
         {/* ACEKknow Intro */}
-        <div class="container">
-          <h2 ><span>ACEKnow Intro</span></h2>
-          <p class="p1 text-center">
+        <div className="container">
+          <h2 className="h2-about"><span>ACEKnow Intro</span></h2>
+          <div className="contianer-about"><div className="front-break1"></div></div>
```
Comment (CatherineZM) [inline frontend/src/Components/AboutPage.js]: I'm confused why is there two empty div here? What's it for?

File: frontend/src/Components/AboutPage.js [hunk 2]
```diff
@@ -9,65 +9,71 @@ class AboutPage extends React.Component {
...
+          <p className="p1-about text-center">
           His title, King of Heroes, is not meant to call him a king who is a hero, but instead implies that he is the king over all heroes. The heroes of various myths are derived from his legend, so his Gate of Babylon possesses all of their Noble Phantasms. Though there are numerous kings with titles such as the King of Knights or the King of Conquerors, he is the only one in all of heaven and earth crowned with the title of "King of All Heroes."
           </p>
         </div>
 
+        <div className="contianer-about"><div className="end-break"></div></div>
```
Comment (CatherineZM) [inline frontend/src/Components/AboutPage.js:23]: Same problem as the one before

File: frontend/src/Components/AboutPage.js [hunk 3]
```diff
@@ -9,65 +9,71 @@ class AboutPage extends React.Component {
...
         {/* ACE Intro */}
-        <div class="container">
-          <h2><span>ACE Intro</span></h2>
-          <br></br>
-          <div class="row align-items-center">
-            <div class="col-lg-6">
+        <div className="container">
+          <h2 className="h2-about"><span>ACE Intro</span></h2>
+
+          <div className="contianer"><div className="front-break"></div></div>
```
Comment (CatherineZM) [inline frontend/src/Components/AboutPage.js]: Same here

File: frontend/src/Components/AboutPage.js [hunk 4]
```diff
@@ -9,65 +9,71 @@ class AboutPage extends React.Component {
...
         {/* Contact Us */}
-        <br></br><br></br>
-        <div class="container">
-          <h2><span>Contact Us</span></h2>
-          <div class="contact-box">
-            <div class="contact-row">
-              <div class="contact-column">
-                <div class="contact-card">
+        <div className="container">
+          <h2 className="h2-about"><span>Contact Us</span></h2>
```
Comment (CatherineZM) [inline frontend/src/Components/AboutPage.js:50]: Also, why is there span inside of h2? It seems unecessary

File: frontend/src/Components/AboutPage.js [hunk 5]
```diff
@@ -9,65 +9,71 @@ class AboutPage extends React.Component {
...
                     Link to XXX
                   </a>
                 </div>
               </div>
-              <div class="contact-column">
-                <div class="contact-card">
+              <div className="contact-column">
+                <div className="contact-card">
                 Other contact information list here XXXXX
                 <br></br>XXX<br></br>XXX
```
Comment (CatherineZM) [inline frontend/src/Components/AboutPage.js]: `<br></br>` could be replaced with `<br/>` for better coding style

**Review by CatherineZM — CHANGES_REQUESTED** [review]
I have made some comments within the js file, please revise and change accordingly in css file.

File: frontend/src/Components/AboutPage.js (hunk 1 above)
Comment (ghost) [reply frontend/src/Components/AboutPage.js]: The front-break1 is line spacing between title "ACEKow Intro" and text paragraph below it. I'll remove outer div.

File: frontend/src/Components/AboutPage.js (hunk 4 above)
Comment (ghost) [reply frontend/src/Components/AboutPage.js:50]: I use span to help build title styel (---ACE---)

Comment (CatherineZM) [reply frontend/src/Components/AboutPage.js:50]: That makes sense. You can resolve this then.

**Review by CatherineZM — APPROVED** [review]

**Pull request merged**